### PR TITLE
Feat add choice content type home page

### PR DESCRIPTION
**Pull request by amine-betari**
@Plopix 

Add the possibility to choose a model
* content type classic (default
* content type empty (with a field to put pure HTML)

### CODE REVIEW

File: bundle/Core/Service/SiteConfiguration.php [hunk 1]
```diff
@@ -52,6 +52,7 @@ public function duplicate(SiteConfigurationEntity $configuration, string $suffix
         $new->setAdminEmail($configuration->getAdminEmail());
         $new->setCreated(new Datetime());
         $new->setGroups($configuration->getGroups());
+        $new->setModele($configuration->getModele());
```
Comment (Plopix) [inline bundle/Core/Service/SiteConfiguration.php]: `setTemplate`? or `setModel`

**Template** is probably the best here.

File: bundle/Core/SiteAccess/Processor.php [hunk 2]
```diff
@@ -104,7 +104,13 @@ function (UserGroup $group) {
             $data['groupIds'] = null;
         }
 
-        $content = $this->twig->render('@ezdesign/site_configuration/siteaccess.yaml.twig', $data);
+        // Check type content type (novaezsiteaccessfactory_home_page_vide or novaezsiteaccessfactory_home_page)
+        if ($configuration->getModele() == 1) {
+            $content = $this->twig->render('@ezdesign/site_configuration/siteaccess_vierge.yaml.twig', $data);
```
Comment (Plopix) [inline bundle/Core/SiteAccess/Processor.php]: `vierge` -> `empty`

File: bundle/Core/SiteAccess/Processor.php [hunk 3]
```diff
@@ -104,7 +104,13 @@ function (UserGroup $group) {
             $data['groupIds'] = null;
         }
 
-        $content = $this->twig->render('@ezdesign/site_configuration/siteaccess.yaml.twig', $data);
+        // Check type content type (novaezsiteaccessfactory_home_page_vide or novaezsiteaccessfactory_home_page)
+        if ($configuration->getModele() == 1) {
```
Comment (Plopix) [inline bundle/Core/SiteAccess/Processor.php]:
```suggestion
        if ($configuration->getModele() === true) {
```

File: documentation/INSTALL.md [hunk 4]
```diff
@@ -83,8 +83,8 @@ novaezsiteaccessfactory_default_siteaccess_groups: ['site_groupe1', 'site_group2
 
 If you use MariaDB, the version of MariaDB must be at least version 10.2
 
-This bundle will add 1 table named: `novaez_siteaccess_factory_site_configuration` and will create 2 Content Types named
-`novaezsiteaccessfactory_home_page` and `novaezsiteaccessfactory_site_configuration`
+This bundle will add 1 table named: `novaez_siteaccess_factory_site_configuration` and will create 3 Content Types named
+`novaezsiteaccessfactory_home_page`, `novaezsiteaccessfactory_home_page_vide` and `novaezsiteaccessfactory_site_configuration`
```
Comment (Plopix) [inline documentation/INSTALL.md]: `vide` ->`empty`

File: bundle/Resources/views/themes/admin/site_configuration/content_types.yaml.twig [hunk 5]
```diff
@@ -124,3 +124,45 @@
             disable-translation: false
             category: 'content'
             position: 70
+
+# Home page Vide
+-
+    type: content_type
+    mode: create
+    content_type_group: 'Content'
+    identifier: novaezsiteaccessfactory_home_page_vide
```
Comment (Plopix) [inline bundle/Resources/views/themes/admin/site_configuration/content_types.yaml.twig]: `vide` -> `empty`

File: documentation/INSTALL.md (hunk 4 above)
Comment (Plopix) [reply documentation/INSTALL.md]: I even wonder of that name is correct, how about: `novaezsiteaccessfactory_static_home_page`

File: documentation/USAGE.md [hunk 6]
```diff
@@ -8,6 +8,9 @@ For now this bundle allows you to configure some properties of a Site Access.
    - Select a **type** and a **theme**, those 2 will be joined with `_` to define the `design`
    - Example: `event` `dark` will setup the design `event_dark` for this Site Access. 
 - **languages**: you can setup the prioritize list of languages
+- **modele**: select content type for home page.
```
Comment (Plopix) [inline documentation/USAGE.md:11]: so no,  that is not true in this PR.
This is a flag, then nobody selects anything.

We must be clear and reflect the business logic.

**Review by Plopix — CHANGES_REQUESTED** [review]
- Careful we code in English.

Also I don't really get the meaning, if it is a "Template" then why is it just `true` or `false` is sounds like it is more about filling the Content Tree or not,

File: bundle/Core/Service/SiteConfiguration.php (hunk 1 above)
Comment (amine-betari) [reply bundle/Core/Service/SiteConfiguration.php]: @Plopix  done

File: bundle/Core/SiteAccess/Processor.php (hunk 2 above)
Comment (amine-betari) [reply bundle/Core/SiteAccess/Processor.php]: @Plopix done

File: documentation/INSTALL.md (hunk 4 above)
Comment (amine-betari) [reply documentation/INSTALL.md]: @Plopix done for changing identifier by novaezsiteaccessfactory_static_home_page

File: bundle/Resources/views/themes/admin/site_configuration/content_types.yaml.twig (hunk 5 above)
Comment (amine-betari) [reply bundle/Resources/views/themes/admin/site_configuration/content_types.yaml.twig]: done for changing identifier by novaezsiteaccessfactory_static_home_page

File: documentation/USAGE.md (hunk 6 above)
Comment (amine-betari) [reply documentation/USAGE.md:11]: done

File: bundle/Core/SiteAccess/Processor.php (hunk 3 above)
Comment (amine-betari) [reply bundle/Core/SiteAccess/Processor.php]: At the second time, we will have other options in the select

**Pull request merged**